starters/docs/src/theme.css

```diff
@@ -1,88 +1,162 @@
+  --tint-1100: oklch(from var(--tint) var(--lightness-1100) var(--chroma-1100) h);
+  --tint-1200: oklch(from var(--tint) var(--lightness-1200) var(--chroma-1200) h);
+  --tint-1300: oklch(from var(--tint) var(--lightness-1300) var(--chroma-1300) h);
+  --tint-1400: oklch(from var(--tint) var(--lightness-1400) var(--chroma-1400) h);
+  --tint-1500: oklch(from var(--tint) var(--lightness-1500) var(--chroma-1500) h);
+  --tint-1600: oklch(from var(--tint) var(--lightness-1600) var(--chroma-1600) h);
+
+  /* gray scale */
+  --gray-100: oklch(from var(--gray) var(--lightness-100) c h);
+  --gray-200: oklch(from var(--gray) var(--lightness-200) c h);
+  --gray-300: oklch(from var(--gray) var(--lightness-300) c h);
+  --gray-400: oklch(from var(--gray) var(--lightness-400) c h);
+  --gray-500: oklch(from var(--gray) var(--lightness-500) c h);
+  --gray-600: oklch(from var(--gray) var(--lightness-600) c h);
+  --gray-700: oklch(from var(--gray) var(--lightness-700) c h);
+  --gray-800: oklch(from var(--gray) var(--lightness-800) c h);
+  --gray-900: oklch(from var(--gray) var(--lightness-900) c h);
+  --gray-1000: oklch(from var(--gray) var(--lightness-1000) c h);
+  --gray-1100: oklch(from var(--gray) var(--lightness-1100) c h);
+  --gray-1200: oklch(from var(--gray) var(--lightness-1200) c h);
+  --gray-1300: oklch(from var(--gray) var(--lightness-1300) c h);
+  --gray-1400: oklch(from var(--gray) var(--lightness-1400) c h);
+  --gray-1500: oklch(from var(--gray) var(--lightness-1500) c h);
+  --gray-1600: oklch(from var(--gray) var(--lightness-1600) c h);
+}
+
+/* light mode colors */
 :root {
   --background-color: #f8f8f8;
   --gray-50: #ffffff;
-  --gray-100: #d0d0d0;
-  --gray-200: #afafaf;
-  --gray-300: #8f8f8f;
-  --gray-400: #717171;
-  --gray-500: #555555;
-  --gray-600: #393939;
-  --purple-100: #d5c9fa;
-  --purple-200: #b8a3f6;
-  --purple-300: #997cf2;
-  --purple-400: #7a54ef;
-  --purple-500: #582ddc;
-  --purple-600: #3c1e95;
-  --red-100: #f7c4ba;
-  --red-200: #f29887;
-  --red-300: #eb664d;
-  --red-400: #de2300;
-  --red-500: #a81b00;
-  --red-600: #731200;
+
+  --lightness-100: 98.1187%;
+  --lightness-200: 95.2045%;
+  --lightness-300: 91.1434%;
+  --lightness-400: 85.1751%;
+  --lightness-500: 79.1773%;
+  --lightness-600: 72.3297%;
+  --lightness-700: 67.0121%;
+  --lightness-800: 62.3039%;
+  --lightness-900: 57.9699%;
+  --lightness-1000: 51.9076%;
+  --lightness-1100: 46.9058%;
+  --lightness-1200: 41.0821%;
+  --lightness-1300: 35.3616%;
+  --lightness-1400: 29.6725%;
+  --lightness-1500: 24.5366%;
+  --lightness-1600: 16.6959%;
+
+  /* lower chroma at low lightness levels */
+  --chroma-100: calc(l * c * 0.5);
+  --chroma-200: calc(l * c * 0.6);
+  --chroma-300: calc(l * c * 0.7);
+  --chroma-400: calc(l * c * 0.8);
+  --chroma-500: calc(l * c * 0.9);
+  --chroma-600: c;
+  --chroma-700: c;
+  --chroma-800: c;
+  --chroma-900: c;
+  --chroma-1000: c;
+  --chroma-1200: c;
```

Comment:
@devongovett was browsing this code for fun and noticed you forgot the 1100 chroma variable here :)